test/runtests.jl

```diff
@@ -44,61 +45,68 @@ setprogress!(false)
 
 include(pkgdir(Turing)*"/test/test_utils/AllUtils.jl")
 
+# Collect timing and allocations information to show in a clear way.
+const to = TimerOutput()
```

Comment:
In that case we have to qualify
```suggestion
const to = TimerOutputs.TimerOutput()
```